src/Entities/CourseEntityDetailed.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Portal\Entities;
+
+class CourseEntityDetailed extends CourseEntity
```

Comment:
this entity name is inconsistent with the other entities in the application. please bring it inline with the others (i.e. `Entity` should be the last word)